scripts/mixpanel_traffic_all.py

```
@@ -0,0 +1,211 @@
+
+    try:
+        token = get_token(client_id, client_secret)
+    except Exception as e:
+        print(f"[{label}] ✗ token failed: {e}")
+        return pub_id, 0
+
+    sent = 0
+    while sent < target:
+        sent += burst(token, profile)
+        # Idle gap between sessions
+        time.sleep(random.uniform(0.2, 1.0))
```

Ruff (0.15.12)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)